app/bt/PermissionsContext.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { createContext, useState, useEffect } from 'react';
+
+import * as BTNativeModule from './nativeModule';
+
+import {
+  checkNotifications,
+  requestNotifications,
+} from 'react-native-permissions';
+import { Platform } from 'react-native';
+import { PermissionStatus, statusToEnum } from '../helpers/PermissionsHelpers';
+
+type ENEnablement = `DISABLED` | `ENABLED`;
+type ENAuthorization = `UNAUTHORIZED` | `AUTHORIZED`;
+
+export type ENPermissionStatus = [ENAuthorization, ENEnablement];
+
+const initialENStatus: ENPermissionStatus = ['UNAUTHORIZED', 'DISABLED'];
+
+interface PermissionContextState {
+  notification: {
+    status: PermissionStatus;
+    check: () => void;
+    request: () => void;
+  };
+  exposureNotifications: {
+    status: ENPermissionStatus;
+    check: () => void;
+    request: () => void;
+  };
+}
+
+const initialState = {
+  notification: {
```

Comment:
we'll be adding UI to request `Notifications` permissions during onboarding soon, so adding this here now